bats/fb-test-backup.bats

```diff
@@ -83,3 +97,15 @@ BACKUP_ONLINE_DIR="${BACKUP_BASE_DIR}/online"
   tFileExists "${BACKUP_ONLINE_DIR}/candlepin.dump"
   tFileExists "${BACKUP_ONLINE_DIR}/pulpcore.dump"
 }
+
+@test "check online backup contents for Foreman Proxy" {
+  tForemanMaintainAvailable
+
+  if tPackageExists foreman; then
+    skip "Stand-alone foreman-proxy test"
+  fi
+
+  tFileExists "${BACKUP_ONLINE_DIR}/pulp_data.tar"
+  tFileExists "${BACKUP_ONLINE_DIR}/pulpcore.dump"
+  tar -tvf "${BACKUP_ONLINE_DIR}/config_files.tar.gz" | grep "${HOSTNAME}.tar.gz"
```

Comment:
```suggestion
  tar -tvf "${BACKUP_ONLINE_DIR}/config_files.tar.gz" | grep "${HOSTNAME}-certs.tar"
```
It should be like this, or at least that's how the file on my machine is.

Reply:
This depends on what you pass to the certs generate command. And our role in forklift uses: `foreman_proxy_content_certs_tar: "/root/{{ ansible_nodename }}.tar.gz"`